### ISSUE TITLE
Remove remaining JavaScript language tests

### ISSUE THREAD

**Issue by jugglinmike** [w3c/web-platform-tests#6462]
In [1], eight tests for the ECMA-262 specification were removed from this
project in deference of the Test262 suite. There still exist a number of tests
for built-ins defined by that specification. These tests should likewise be
removed, and in cases where Test262 does not have equivalent coverage, the
missing tests should be introduced there.

[1] https://github.com/w3c/web-platform-tests/pull/5127

**Comment by foolip** [comment]
@littledan FYI, these tests exist because adding them here was easier than contributing to test262. Right, @Ms2ger?

**Comment by Ms2ger** [comment]
Yeah. Things might be better now, but at least when I wrote them contributing to test262 was painful enough that I gave up.

**Comment by littledan** [comment]
I dunno, contributing to test262 isn't perfect, but there are some things that are just inherently slightly different when you want to write a test against a JS shell vs a browser. These tests require a browser so they will be more annoying to use for JS engine authors.

@ms2ger what kind of pain did you run into? We are actually discussing potential improvements [here](https://github.com/tc39/test262/issues/1258)
